src/Commands/MakeMailCommand.php

```diff
@@ -31,7 +31,7 @@ public function handle(): void
 
         $namespace = Str::of(string: 'Mail')
             ->start(prefix: $this->resolveDomainArgument() . '\\')
```

Comment:
this one not able to use resolveNamespace?

Reply:
it infrastructure, i still don't have idea to support infrastructure for that method

Reply:
alright later we refactor if found the better way